docs/csharp/programming-guide/concepts/linq/toc.yml

```diff
@@ -144,7 +138,6 @@
         - name: "How to: Build LINQ to XML Examples"
           href: how-to-build-linq-to-xml-examples.md
       - name: Creating XML Trees
```

Comment:
I would name this "XML trees". The child topics are not only about creating, but parsing and processing.

Also, on line 147, did you mean to leave parsing-xml.md or delete it? It's just a portal page, and you've changed links to it elsewhere. (Or maybe that's done in the next PR?)

Reply:
`parsing-xml.md` is removed in the next PR.